inc/sitemaps-index.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Class Core_Sitemaps_Index.
+ * Builds the sitemap index page that lists the links to all of the sitemaps.
+ *
+ * @todo This will probably be split out so that rewrites are in a class, building the xml output is a class,
+ * rendering sitemaps content is a class etc.
+ */
+class Core_Sitemaps_Index {
+
+	/**
+	 * Content of the sitemap to output.
+	 *
+	 * @var string
+	 */
+	protected $sitemap_content = '';
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'url_rewrites' ), 99 );
```

Comment:
Let's move actions/filters out of the constructor and into a separate `bootstrap()` method. Even though you will find this pattern common in WordPress code, I generally consider it an anti-pattern that should be avoided so that you can instantiate the class without introducing side effects. It also makes your code more testable because you can load your class without hooking into all the WordPress functionality.